pytgcalls/pytgcalls/group_call_native.py

```diff
@@ -350,7 +353,10 @@ async def start(
             raise RuntimeError('Chat without a voice chat')
 
         if join_as is None:
-            self.join_as = self.my_peer
+            if self.default_join_as:
+                self.join_as = self.default_join_as
```

Comment:
`400 PEER_ID_INVALID` when we use default_join_as in join method as join_as param